katello/lib/backup.rb

```diff
@@ -0,0 +1,420 @@
+require 'optparse'
+require 'fileutils'
+require 'date'
+require 'yaml'
+require 'find'
+require 'json'
+require 'highline/import'
+require "/usr/share/katello/script-helper.rb"
+
+class Backup
+  include ScriptHelper
+
+  def initialize(foreman_proxy_content, foreman_proxy)
+    @excluded="--exclude goferd,foreman-proxy,squid,smart_proxy_dynflow_core,qdrouterd,qpidd"
```

Comment:
I know you haven't changed this but doesn't qpidd have to stop to get a consistent backup of `/var/lib/qpidd` ?

Reply:
Thanks @sean797, I marked it WIP as I do still have that as a change to make, but wanted to get eyes on this early. 

I'm not sure about qpidd  stopping to get an accurate backup, maybe @jlsherrill or @ehelms can comment